docs/meetups/index.rst

```diff
@@ -0,0 +1,55 @@
+Meetups
+=======
+
+**Write the Docs Meetups** allow for documentarians to get to know their
+local community better. Meeting in person is a great way to stay up to
+date with the documentation world, and meet some great people in the
+process.
+
+All of our local meetups are organized by members of the community in
+their own town. They vary in style as much as the individual cities
+themselves vary. The main thread that brings people together is caring
+about great communication and users of their product.
+
+We invite you to join a meetup in your local community. If your local community
+isn't listed here, feel free to `start one <../organizer-guide/meetups/starting>`_
+and let us know. We'd be happy to promote it and list it here.
+
+
+Current Meetups: North America
+------------------------------
+
+.. meetup-listing::
+    :region: North America
+
+
+Current Meetups: Europe
+-----------------------
+
+.. meetup-listing::
+    :region: Europe
+
+
+Current Meetups: Oceania
+------------------------
+
+.. meetup-listing::
+    :region: Oceania
+
+
+Current Meetups: Asia
+---------------------
+
+.. meetup-listing::
+    :region: Asia
```

Comment:
I'm unsure. People don't have to edit the index, just add a single YAML with fairly straightforward info.

@zerok and I did discuss the added complexity. 

The reason to move towards structured meetups is that it lets us add meetup videos to them, and iterate over the same list.